activities.json

```
@@ -1050,6 +1059,12 @@
     "ciuName": "midi",
     "description": "This specification defines an API supporting the MIDI (Musical Instrument Digital Interface) protocol, enabling web applications to enumerate and select MIDI input and output devices on the client system and send and receive MIDI messages. It is intended to enable non-music MIDI applications as well as music ones, by providing low-level access to the MIDI devices available on the users' systems.",
     "id": "webmidi",
+    "mdnUrl": [
+      "https://developer.mozilla.org/en-US/docs/Web/API/MIDIAccess",
+      "https://developer.mozilla.org/en-US/docs/Web/API/MIDIInput",
+      "https://developer.mozilla.org/en-US/docs/Web/API/MIDIConnectionEvent",
+      "https://developer.mozilla.org/en-US/docs/Web/API/MIDIInputMap"
+    ],
```

```suggestion
    "mdnUrl": "https://developer.mozilla.org/en-US/docs/Web/API/MIDIAccess",
```

I think that this is the top-level API, and if it isn't, that's OK.  https://developer.mozilla.org/en-US/docs/Web/API/Web_MIDI_API doesn't exist yet, which would have been better.